### ISSUE TITLE
[SUGGESTION] Save Editor

### ISSUE THREAD

**Issue by Paradox-Edge** [CiarenceW/CiarenceUnbelievableModifications#3]
This is probably going to be quite the tall ask. But if you could put in a save editor that lets you unlock/lock anything and everything that can be unlocked in the game, that would be very nice. I 100%ed the game in the past but got a new computer, realizing only now as of making this post that there's no cloud save for the game. I _really_ don't want to have to go through the grueling task of having to re-unlock everything in the game again.

I'm talking things like the guns, achievements, collectibles, challenges at the compound, general secrets at the compound, you name it, it should be in the editor.

Please. 🙏

**Comment by CiarenceW** [comment]
Yeah maybe I'll get to around to doing something like that in the future, wouldn't be that hard I don't think.

**Comment by Paradox-Edge** [comment]
I could paypal you like 200 bucks for you to do it now instead of maybe sometime in the future? To give you an incentive.

**Comment by CiarenceW** [comment]
uh yeah sure why not, add me on discord for the paypal stuff (it's ciarencew), for the save editor part, I was thinking you'd open by clicking a button in the debug menu, and then you could choose which stuff to unlock (or everything)
